main.go

```diff
@@ -123,6 +125,32 @@ func main() {
 			return nil
 		})
 
+	flag.Func("cis-detail-report-labels",
+		"A comma-separated list of labels to be exposed per-report. Alias 'all' is supported.",
+		func(input string) error {
+			items := strings.Split(input, ",")
+			for _, i := range items {
+				if i == ciskubebenchreport.LabelGroupAll {
+					// Special case for "all".
+					cisReportLabels = appendIfNotExistsCIS(cisReportLabels, ciskubebenchreport.LabelsForGroup(ciskubebenchreport.LabelGroupAll))
+					continue
+				}
+
+				label, ok := ciskubebenchreport.LabelWithName(i)
+				if !ok {
+					err := errors.New("invalidConfigError")
+					return err
+				}
+				cisReportLabels = appendIfNotExistsCIS(cisReportLabels, []ciskubebenchreport.ReportLabel{label})
+			}
+
+			// If exposing detail metrics, we must always include the report name in order to delete them by name later.
+			reportNameLabel, _ := ciskubebenchreport.LabelWithName("report_name")
+			cisReportLabels = appendIfNotExistsCIS(cisReportLabels, []ciskubebenchreport.ReportLabel{reportNameLabel})
+
+			return nil
+		})
+
```

Comment:
Below this on line 175 there is a place which prints the custom labels used for vulnerabilityreports. It would be nice to add/update that so that it differentiates between the vulnerabilityreport target labels and the cis target labels

Reply:
Missed it, added 👍